packages/devp2p/src/dpt/dpt.ts

```diff
@@ -55,6 +55,7 @@ export class DPT extends EventEmitter {
   private _kbucket: KBucket
   private _server: DPTServer
   private _refreshIntervalId: NodeJS.Timeout
+  private _refreshIntervalSelectionCounter: number = 0
```

Comment:
same nit here for `number` type annotation, I believe not needed when already initialized with a number